src/main/java/uk/gov/hmcts/reform/ccd/document/am/model/CaseDocumentsMetadata.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.reform.ccd.document.am.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class CaseDocumentsMetadata {
+
+    @JsonProperty
```

Comment:
There is a @Jacksonized lombok annotation is available now. That eliminates need of @JsonProperty and other lombok annotations like @NoArgsConstructor etc

Reply:
`@Jacksonized was introduced as experimental feature in lombok v1.18.14.`

Are we ok with using experimental features ?

Reply:
That's introduced almost a year ago.. if our tests are passing then i think it's OK.